### ISSUE TITLE
Domains routing?

### ISSUE THREAD

**Issue by ghost** [kbrsh/coat#2]

**Comment by kbrsh** [comment]
What do you mean?

**Comment by ghost** [comment]
I mean, domains support.

**Comment by kbrsh** [comment]
Uhh, Coat already routes all paths to any domain. Also, it's still a work in progress and freezes sometimes, I wouldn't recommend using it until I formally release it.

**Comment by ghost** [comment]
But only to one domain

**Comment by kbrsh** [comment]
You're right--Coat will support multiple domains and load balancing...soon.

**Comment by ghost** [comment]
Ok. I'm waiting.
